dev/io.openliberty.microprofile.reactive.messaging30.internal_fat_tck/fat/src/io/openliberty/microprofile/reactive/messaging30/tck/ReactiveMessagingTCKLauncher.java

```diff
@@ -48,12 +48,12 @@ public static void setUp() throws Exception {
 
     @AfterClass
     public static void tearDown() throws Exception {
-        server.stopServer("CWWKZ000[24]E"); // Ignore app start errors - there are lots of tests for invalid apps
+        server.stopServer("CWWKZ000[24]E", "CWMRX1100E"); // Ignore app start errors - there are lots of tests for invalid apps
     }
 
     @Test
     @Mode(TestMode.FULL)
-    @AllowedFFDC({ "org.jboss.weld.exceptions.DeploymentException", "com.ibm.ws.container.service.state.StateChangeException" }) // The tested deployment exceptions cause FFDC so we have to allow for this.
+    @AllowedFFDC({ "org.jboss.weld.exceptions.DeploymentException", "com.ibm.ws.container.service.state.StateChangeException", "jakarta.enterprise.inject.spi.DeploymentException" }) // The tested deployment exceptions cause FFDC so we have to allow for this.
```

Comment:
Not sure why you need to add `jakarta.enterprise.inject.spi.DeploymentException` - I _think_ you'd only need that if someone was creating an instance of exactly that type and throwing it, whereas now you're just rethrowing the `DeploymentException` generated by smallrye RM, and that didn't need this previously.

Reply:
Can confirm having rerun, we now do get `jakarta.enterprise.inject.spi.DeploymentException` FFDCS, coming from the smallrye code 
```
Stack Dump = jakarta.enterprise.inject.spi.DeploymentException: Wiring error(s) detected in application.
	at io.smallrye.reactive.messaging.providers.extension.MediatorManager.start(MediatorManager.java:243)
	at io.smallrye.reactive.messaging.providers.extension.MediatorManager$Proxy$_$$_WeldClientProxy.start(Unknown Source)
```

Looking at the Smallrye code, this is what we should of been getting as they do throw the `Jakarta` version, so it is fact that we are now catching this earlier in app starting process that we get the `Jakarta` FFDC as well as the WELD one.